packages/graphql/lib/src/links/websocket_link/websocket_client.dart

```diff
@@ -222,7 +227,7 @@ class SocketClient {
       // Even though config.connect is sync, we call async in order to make the
       // SocketConnectionState.connected attribution not overload SocketConnectionState.connecting
       socketChannel =
-          await config.connect(Uri.parse(url), protocols).forGraphQL();
+          (await config.connect(Uri.parse(url), protocols)).forGraphQL();
```

Comment:
```
var connection = await config.connect(Uri.parse(url), protocols)
suggestion = connection.forGraphQL();
```

This is a more clean way to write the same code, but it is more accessible for people people with disabilities